lib/std/net.zig

```diff
@@ -10,15 +10,19 @@ test "" {
     _ = @import("net/test.zig");
 }
 
-pub const IpAddress = extern union {
+pub const Address = extern union {
     any: os.sockaddr,
     in: os.sockaddr_in,
     in6: os.sockaddr_in6,
+    un: switch (builtin.os) {
+        .linux, .macosx, .freebsd, .netbsd => os.sockaddr_un,
```

Comment:
maybe instead of this switch we could have in this scope:

```zig
const have_unix_sockets = @hasDecl(os, "sockaddr_un");
```

so then the field can be: `un: if (have_unix_sockets) os.sockaddr_un else void,`. And elsewhere this switch can be avoided as well in favor of `have_unix_sockets`.